README.md

```diff
@@ -81,7 +81,7 @@ Please **do not submit a pull request to, or open any issues in this repo**. Thi
   - Clicking the "Use this template" button on the GitHub home page
   - Visiting the generate page: [mktg-webdev-challenge/generate](https://github.com/hashicorp/mktg-webdev-challenge/generate)
 - In your new repository, go to Settings > Collaborators
-- Add the following usernames as collaborators: `mwickett`, `dstaley`, `kaitlynnefuery` and `zchsh`
+- Add the following usernames as collaborators: `mwickett`, `dstaley`, `kaitlynnefuery`, `zchsh`, and `rubensandwich`.
```

Comment:
who doesn't love an oxford comma? (besides the AP style, which IMO is objectively a wrong decision.)